src/components/Apps/AssetSelect/dialog.vue

```diff
@@ -82,7 +82,7 @@ export default {
           },
           {
             prop: 'address',
-            label: this.$t('IPDomain'),
+            label: this.$t('Address'),
             sortable: 'custom'
           },
           {
```

Comment:
I'm unable to analyze the specific differences mentioned here without more context. Please provide details about which parts of the code you wish to have checked, such as the structure of objects they belong to, their methods involved, or even just the names and types of properties listed (which would be useful for any comparisons).